test/index-test.js

```diff
@@ -1,71 +0,0 @@
-/* eslint-disable no-unused-expressions, max-nested-callbacks */
-import {expect} from 'chai';
-import {default as index} from '../src/index.js';
-
-describe('The formatted stream module', () => {
-  it('should export a `guessFormat` function', () => {
-    expect(index.guessFormat).to.be.a('function');
-  });
-
-  it('should export a `getKnownFormats` function', () => {
-    expect(index.getKnownFormats).to.be.a('function');
-  });
-
-  it('should export a `from` function', () => {
-    expect(index.from).to.be.a('function');
-  });
-
-  it('should export a `to` function', () => {
-    expect(index.to).to.be.be.a('function');
-  });
-
-  describe('.guessFormat', () => {
-    it('should return \'csv\' for csv files', () => {
-      const format = index.guessFormat('/home/data/myfile.csv');
-      expect(format).to.equal('csv');
-    });
-
-    it('should return \'xlsx\' for xlsx files', () => {
-      const format = index.guessFormat('myfile.xlsx');
-      expect(format).to.equal('xlsx');
-    });
-
-    it('should return \'json\' for json files', () => {
-      const format = index.guessFormat('./data.json');
-      expect(format).to.equal('json');
-    });
-  });
-
-  describe('.getKnownFormats', () => {
-    it('should list \'csv\', \'xlsx\', and \'json\'', () => {
-      const knownFormats = index.getKnownFormats(),
-        expectedKnownFormats = [
-          'csv',
-          'xlsx',
-          'json'
-        ];
```

Comment:
Then make it individual tests — not sure what is best practice. In a client project we have nested loops to test SVG icon generation. Doing that by hand would be very intensive.